annotation/app/errors.py

```diff
@@ -156,5 +156,5 @@ def agreement_score_service_error_handler(
 ):
     return JSONResponse(
         status_code=400,
-        content={"detail": f"Agreement score service error. {exc.exc_info}"},
+        content={"detail": f"Agreement score service error. {exc}"},
     )
```

Comment:
I usually do:
try:
    ...
except SomeException as exc:
    raise HTTPException(status_code=400, detail="Agreement score service error. {exc}"

But it's up to you

Reply:
This is a common error handler